### ISSUE TITLE
Mongo DB Testing Error

### ISSUE THREAD

**Issue by Fenikkusu** [phalcon/incubator#838]
When running the codeception unit suite on my laptop, this error is coming up: 

```
ValidationCest: Test collection validation
 Test  tests/unit/Mvc/Collection/ValidationCest.php:mongo
                                                                           
  [RuntimeException] Call to undefined method UnitTester::markTestSkipped  
                                                                           
#1  .../PhpstormProjects/phalcon-incubator/tests/_support/Helper/CollectionTrait.php:23
#2  .../PhpstormProjects/phalcon-incubator/tests/unit/Mvc/Collection/ValidationCest.php:53
#3  Phalcon\Test\Mvc\Collection\ValidationCest->mongo
```

I've ran the codecept build command, but this had no affect. I don't have mongo and don't plan on installing it. I'm not sure of the 'codeception' method to fixing this and I'm currently working on something else, so I wanted to go ahead and report it at the least. In most cases, the tests extend Unit. In this one case it's a Cest instead, and 'markTestSkipped' isn't there, unless I've configured my yaml's incorrectly...

**Comment by sergeyklay** [comment]
Probably we should replace `markTestSkipped` by
 
```php
throw new \PHPUnit_Framework_SkippedTestError(' ... ');
```

**Comment by Fenikkusu** [comment]
I actually tried that.

```
        if (!extension_loaded('mongodb')) {
            throw new \PHPUnit_Framework_SkippedTestError(
                'Warning: mongo extension is not loaded'
            );
        }
```

The problem was that after I did that, PHPStorm started acting weird and terminated the Codeception run. It seemingly ran alright in the terminal, but I wasn't sure if it was really the proper fix or not. Granted, I've had troubles before with the codeception integration.

**Comment by sergeysviridenko** [comment]
@Fenikkusu I can't reproduce. Actually you can run not all tests. You can separate and use just you test if you'd like to test something yours.

**Comment by sergeyklay** [comment]
Fixed in the `3.2.x` branch. Feel free to open a new issue if the problem appears again. Thank you for contributing.
